### ISSUE TITLE
open_in_browser() failing silently

### ISSUE THREAD

**Issue by dtyulman** [nilearn/nilearn#1746]
On my system (macOS 10.12.6), the following code does not open the corresponding html file in the browser (tried both Firefox and Chrome):
```
markers = plotting.view_markers(xyz, colors)
markers.open_in_browser()
```
Prepending `'file://'` to the `file_name` string e.g. `webbrowser.open('file://{}'.format(file_name))` seems to fix the issue for me.

**Comment by bthirion** [comment]
Can you issue a PR ?
Thx.

**Comment by GaelVaroquaux** [comment]
Fixed in #1747
